integration-tests/src/test/java/oracle/kubernetes/operator/ItManagedCoherence.java

```diff
@@ -0,0 +1,338 @@
+            domainMap.put("clusterName", "appCluster");
+            domainMap.put("initialManagedServerReplicas", new Integer("2"));
+            domainMap.put("customDomainTemplate", customDomainTemplate);
+            domainMap.put(
+                    "createDomainPyScript",
+                    "integration-tests/src/test/resources/domain-home-on-pv/"
+                            + COHERENCE_CLUSTER_SCRIPT);
+            if ((System.getenv("LB_TYPE") != null && System.getenv("LB_TYPE").equalsIgnoreCase("VOYAGER"))
+                    || (domainMap.containsKey("loadBalancer")
+                    && ((String) domainMap.get("loadBalancer")).equalsIgnoreCase("VOYAGER"))) {
+                domainMap.put("voyagerWebPort", new Integer("30366"));
+            }
+            domain = TestUtils.createDomain(domainMap);
+            domain.verifyDomainCreated();
+            String[] pods = {
+                    DOMAINUID + "-" + domain.getAdminServerName(),
+                    DOMAINUID + "-managed-server",
+                    DOMAINUID + "-managed-server1",
+                    DOMAINUID + "-managed-server2",
+                    DOMAINUID + "-new-managed-server1",
+                    DOMAINUID + "-new-managed-server2",
+            };
+            verifyServersStatus(domain, pods, DOMAINUID);
+
+            // Build WAR in the admin pod and deploy it from the admin pod to a weblogic target
+            TestUtils.buildDeployCoherenceAppInPod(domain,
+                    testAppName, scriptName, BaseTest.getUsername(), BaseTest.getPassword(),
+                    appToDeploy, "dataCluster");
+
+            coherenceCacheTest();
+
+            testCompletedSuccessfully = true;
+        } finally {
+            if (domain != null && !SMOKETEST && (JENKINS || testCompletedSuccessfully)) {
+              domain.destroy();
+            }
+        }
+        logger.info("SUCCESS - " + testMethodName);
+    }
+
+    /**
+     * Create operator and verify its deployed successfully. Create domain with 2 Managed coherence
+     * clusters verify domain is started. Deploy an application to the cluster with no storage enabled
+     * and the GAR file to the cluster with storage enabled. Verify that data can be added and stored
+     * in the cache and can also be retrieved from cache.
+     *
+     * @throws Exception exception
+     */
+    @Test
+    public void testCreateCoherenceDomainInImageUsingWlst() throws Exception {
+        Assume.assumeFalse(QUICKTEST);
+        String testMethodName = new Object() {}.getClass().getEnclosingMethod().getName();
+
+        logTestBegin(testMethodName);
+        logger.info("Creating coeherence domain in image using wlst and testing the cache");
+
+        boolean testCompletedSuccessfully = false;
+        domain = null;
+
+        // create operator1
+        if (operator1 == null) {
+            operator1 = TestUtils.createOperator(OPERATOR1_YAML);
+        }
+
+        try {
```

Comment:
looks like alot of common code with testCreateCoherenceDomainOnPvUsingWlst could be shared